cli/util/apiinfo.go

```diff
@@ -36,6 +37,31 @@ func ParseApiInfo(s string) APIInfo {
 	}
 }
 
+func ParseApiInfoMulti(s string) []APIInfo {
+	var apiInfos []APIInfo
+
+	if infoWithToken.Match([]byte(s)) {
```

Comment:
How does this work if we didn't update the `infoWithToken` regex above?

Reply:
I'm not sure. Do you expect it to fail?

Reply:
Yeah, I'm a little confused why this method works with multiple API infos. I'm assuming it's exercised in your manual testing.

Reply:
yep. I set the FULLNODE_API_INFO env var for the miner